src/navigators/RootStack.tsx

```diff
@@ -25,22 +46,42 @@ export function RootStack() {
   }
 
   if (isLoggedIn) {
-    if (loadingConsents) {
+    const Stack = createNativeStackNavigator<LoggedInRootParamList>();
+    const hasAccountAndProject = !!(activeProject?.id && account?.id);
+
+    const initialRoute = !hasAccountAndProject
+      ? 'InviteRequired'
+      : shouldRenderConsentScreen
+      ? 'screens/ConsentScreen'
+      : 'app';
+
+    if (loadingAccountOrProject && initialRoute === 'InviteRequired') {
       return (
         <ActivityIndicatorView
-          message={t('root-stack-waiting-for-consent', 'Waiting for consent')}
+          message={t('root-stack-waiting-for-account-and-project', 'Loading')}
```

Comment:
Why take these arguably more helpful loading messages away?  With this `Loading` matching line 69, we won't be able to visually tell which the app is waiting for (outside of logs).

Reply:
I see, I hadn't realized these messages actually appear in the UI after a timeout. Given that you are totally right, I'll update them.

Reply:
I'd like to get see this in a customer build asap, so merging and one of us can follow up on those loading messages later 👍